.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 Cargo.lock
+kcov_output/
```

Comment:
Can you add a PR in crate-template as well to improve the .gitignore there for future crates that we might add to rust-vmm?